hail/src/test/scala/is/hail/TestUtils.scala

```diff
@@ -352,6 +354,16 @@ object TestUtils {
                     pw.print(s"/* JVM bytecode dump for IR:\n${Pretty(x)}\n */\n\n")
                     pw
                   })
+            case ExecStrategy.JvmCompileUnoptimized =>
+              assert(Forall(x, node => node.isInstanceOf[IR] && Compilable(node.asInstanceOf[IR])))
+              eval(x, env, args, agg, bytecodePrinter =
+                Option(HailContext.getFlag("jvm_bytecode_dump"))
+                  .map { path =>
+                    val pw = new PrintWriter(new File(path))
+                    pw.print(s"/* JVM bytecode dump for IR:\n${Pretty(x)}\n */\n\n")
+                    pw
+                  },
+                optimize = false)
```

Comment:
Here

Reply:
This is a call to TestUtils.eval, not Compile?

Reply:
Compile is called in eval

Reply:
I believe

Reply:
But eval worked before, and you didn't change eval.